nodist/code_coverage

```diff
@@ -1,405 +0,0 @@
-            globf = re.sub(r'\*$', '', globf)
-            exc += " '" + globf + "'"
-        if inc != '':
-            inc = "--include " + inc
-        if exc != '':
-            exc = "--exclude " + exc
-        run("cd " + cc_dir + " ; " + RealPath + "/fastcov.py -C app_total.info " + inc + " " +
-            exc + " -x --lcov -o app_total_f.info")
-        ci_fold_end()
-
-    if Args.stage_enabled[17]:
-        ci_fold_start("report")
-        print("Stage 17: Create HTML")
-        run("cd " + cc_dir + " ; genhtml app_total_f.info --demangle-cpp" +
-            " --rc lcov_branch_coverage=1 --rc genhtml_hi_limit=100 --output-directory html")
-        ci_fold_end()
-
-    if Args.stage_enabled[18]:
-        ci_fold_start("upload")
-        print("Stage 18: Upload")
-        # curl -Os https://cli.codecov.io/latest/linux/codecov ; sudo chmod +x codecov
-        # --disable-search does not seem to work
-        # -d with false directory does not seem to work
-        # So, remove gcno files before calling codecov
-        upload_dir = "nodist/obj_dir/upload"
-        os.makedirs(upload_dir, exist_ok=True)
-        cmd = "ci/codecov -v upload-process -Z" + " -f " + cc_dir + "/app_total.info )"
-        print("print: Not running:")
-        print("  export CODECOV_TOKEN=<hidden>")
-        print("  find . -name '*.gcno' -exec rm {} \\;")
-        print("  " + cmd)
-        ci_fold_end()
-
-    if Args.stage_enabled[19]:
-        print("*-* All Finished *-*")
-        print("")
-        print("* See report in " + cc_dir + "/html/index.html")
-        print("* Remember to make distclean && ./configure before working on non-coverage")
-
-
-def clone_sources(cc_dir):
-    excluded_lines = 0
-    excluded_br_lines = 0
-    for globf in Source_Globs:
-        for infile in glob.glob(globf):
-            if re.match(r'^/', infile):
-                sys.exit("%Error: source globs should be relative not absolute filenames, " +
-                         infile)
-            outfile = cc_dir + "/" + infile
-            outpath = re.sub(r'/[^/]*$', '', outfile, count=1)
-            os.makedirs(outpath, exist_ok=True)
-            with open(infile, "r", encoding="utf8") as fh:
-                with open(outfile, "w", encoding="utf8") as ofh:
-                    lineno = 0
-                    for line in fh:
-                        lineno += 1
-                        line = line.rstrip()
-                        done = False
-                        if re.search(r'LCOV_EXCL_LINE', line):
-                            line += " LCOV_EXCL_BR_LINE"
-                            done = True
-                        elif re.search(r'LCOV_EXCL_START', line):
-                            line += " LCOV_EXCL_BR_START"
-                            done = True
-                        elif re.search(r'LCOV_EXCL_STOP', line):
-                            line += " LCOV_EXCL_BR_STOP"
-                            done = True
```

Comment:
Does the toolage now handle this properly?  Did you compare all reports under old and new system to make sure all old disables like this, plus the ones in the .dat file were still properly suppressed?